--- meta-java/recipes-core/openjdk/openjdk-11-release.inc	2023-10-10 16:13:32.051867288 -0500
+++ meta-java/recipes-core/openjdk/openjdk-11-release.inc	2023-10-10 16:16:36.352103600 -0500
@@ -1,7 +1,7 @@
 # Copyright (C) 2020 Richard Leitner <richard.leitner@skidata.com>
 # Released under the MIT license (see COPYING.MIT for the terms)
 
-SRC_URI[jdk.sha256sum] = "c0d9129adefc033f62cdf57730ab10d38b3f238d7f72e09be89944e58c0e7d1c"
+SRC_URI[jdk.sha256sum] = "c959cca5a8d25e541f40b3d078fa9f82e36fc83aca7cd3633f10cadda9b9a14f"
 
 FILESPATH =. "${FILE_DIRNAME}/patches-openjdk-11:"
 
